libpod/adapter/client.go

```diff
@@ -4,11 +4,19 @@ package adapter
 
 import (
 	"github.com/varlink/go/varlink"
+	"os"
```

Comment:
We try to organize our imports where the golang proper ones come first and then the vendor'd ones ...

os

github.com/...